### PR TITLE
Change "Login or Register" button

### PR DESCRIPTION
**Pull request by caitmuenster**
Changes copy of "Login or Register" button to "Submit/Manage Extension". 

Ref #138

### CODE REVIEW

File: _includes/landing-header.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <div class="cell small-12">
             <h3 class="logo"><a href="{{ "/" | relative_url }}">Extension Workshop</a></h3>
             <nav class="site-nav">
-                <a href="https://addons.mozilla.org/developers/addons" class="button">Register or Log in</a>
+                <a href="https://addons.mozilla.org/developers/addons" class="button">Manage&#47;Submit Extension</a>
```
Comment (muffinresearch) [inline _includes/landing-header.html]: You can use a literal `/` here rather than an entity.

Comment (caitmuenster) [reply _includes/landing-header.html]: 👍

**Comment by muffinresearch** [comment]
@caitmuenster there's one more button with the same text here: https://github.com/mozilla/extension-workshop/blob/acb2ba317c7ede8e010ce33dced87dfb823021f0/_layouts/home.html#L33

**Comment by caitmuenster** [comment]
@muffinresearch Ah, sorry -- this should be covered by this PR: https://github.com/mozilla/extension-workshop/pull/141

**Pull request merged**